content/changelog/2024-04-05-console.md

```diff
@@ -0,0 +1,48 @@
+---
+description: New monitoring widget, support for Local File Cache statistics, plus a teaser for upcoming Organization features
+---
+
+### New Monitoring widget on the Dashboard
+
+We've added a **Monitoring** widget to the Neon **Dashboard**, giving you an easy way to check CPU and RAM usage for your Neon projects. The display defaults to your primary branch but you can switch to view usage details for any branch in your project.
+
+![monitoring widget](/docs/relnotes/monitoring_widget.png "no-border")
+
+We're committed to improving observability for your Neon projects and databases. To learn more about monitoring in Neon, see [Monitoring](/docs/introduction/monitoring).
+
+### A teaser for organizations
+
+You might notice a new addition to our breadcrumb navigation: **Create organization** under your account selector. It's greyed out and inactive for now, but we've included it here as a reminder: organization support is coming soon!
+
+![organization teaser](/docs/relnotes/org_teaser.png "no-border")
+
+Check our [roadmap](/docs/introduction/roadmap) to see what else is coming next.
+
+### PostgreSQL Partition Manager extension
+
+Neon now supports the [pg_partman](/docs/extensions/pg_partman) Postgres extension, which is a partition manager extension that enables creating and managing time and number-based table partition sets in Postgres.
+
+### Local File Cache (LFC) statistics and working set size function
+
+You can now view Local File Cache (LFC) statistics for your Neon compute. The statistics include a cache hit ratio, which can help you determine if your data is accessed from memory or the Neon storage layer.
+The Local File Cache (LFC) stores frequently accessed data in the local memory of your Neon compute instance. The cache helps to reduce latency and improve query performance by minimizing fetches from the Neon storage layer. The LFC acts as an add-on or extension of Postgres [shared buffers](#shared-buffers). In Neon, the `shared_buffers` setting is always 128 MB, regardless of compute size. The LFC extends cache memory up to 80% of your compute's RAM.
+Local File Cache (LFC) statistics are exposed through a `neon_stat_file_cache view`, which you access by installing the `neon` extension.
+
+```sql
+CREATE EXTENSION neon;
```

Comment:
users do not need to (and should not) install this extensions themselves. they should switch to the postgres database and the neon extension should already be available.

psql:
```
\c postgres
```
or directly connect to the postgres database (replace the default neondb with postgres)

Reply:
> (and should not)
I am not sure why you say they should not @skyzh 
It is not necessary because they can connect to another database, but if you want to report those metrics from within a given connection (e.g. in your app) it is still convenient if you can do so within your current connection connected to another database. In that use case it makes perfect sense to install the extension AFAIK

Reply:
I revised it to mention both options. You can't select the postgres database in the SQL Editor in the console. That's another reason for a user to install the extension to their database.

Reply:
Yep sounds good, makes sense for the users to create it in their own database if they want to access the metrics there.